plugin.video.arloview/resources/media/placeholder.txt

```diff
@@ -0,0 +1 @@
+This is where the snapshot images are saved.
```

Comment:
With the above change (`getAddonInfo('profile')`) this isn't true.

Reply:
Maybe you want to remove this file, as it isn't true with your last update anymore.

Reply:
Done.